cate/ds/esa_cci_odp.py

```diff
@@ -154,7 +156,7 @@ def find_datetime_format(filename: str) -> Tuple[Optional[str], int, int]:
 
 async def _extract_metadata_from_odd_url(session=None, odd_url: str = None) -> dict:
     if session is None:
-        session = aiohttp.ClientSession()
+        session = aiohttp.ClientSession(headers={'User-Agent': default_user_agent()})
```

Comment:
Please extract function / method `aiohttp.ClientSession(headers={'User-Agent': default_user_agent()})` to avoid code duplication.